bin/schedule-hooks.js

```diff
@@ -1,87 +0,0 @@
-#!/usr/bin/env node
-var base        = require('taskcluster-base');
-var data        = require('../hooks/data');
-var debug       = require('debug')('hooks:schedule-hooks');
-var path        = require('path');
-var Promise     = require('promise');
-var Scheduler   = require('../hooks/scheduler');
-var taskcreator = require('../hooks/taskcreator');
-
-/* Launch schedule-hooks */
-var launch = async function(profile, options) {
-  options = options || {};
-
-  debug("Launching with profile: %s", profile);
-
-  // Load configuration
-  var cfg = base.config({
-    defaults:  require('../config/defaults'),
-    profile:   require('../config/' + profile),
-    envs: [
-      'pulse_username',
-      'pulse_password',
-      'taskcluster_credentials_clientId',
-      'taskcluster_credentials_accessToken',
-      'azure_accountName',
-      'azure_accountKey',
-      'hooks_tableSigningKey',
-      'hooks_tableCryptoKey',
-      'hooks_publishMetaData',
-    ],
-    filename:  'taskcluster-hooks'
-  });
-
-  // Create Hooks table
-  var Hook = data.Hook.setup({
-    account:      cfg.get('hooks:azureAccount'),
-    table:        cfg.get('hooks:hookTableName'),
-    credentials:  cfg.get('taskcluster:credentials'),
-    authBaseUrl:  cfg.get('taskcluster:authBaseUrl'),
-    signingKey:   cfg.get('hooks:tableSigningKey'),
-    cryptoKey:    cfg.get('hooks:tableCryptoKey'),
-    process:      'schedule-hooks'
-  });
-
-  // Create a validator
-  debug("Waiting for resources to be created");
-  await Hook.ensureTable();
```

Comment:
Actually, when using sas for azure table, issued by auth we don't need to ensure table... In fact we can't...